aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -628,6 +630,43 @@ public class AwsProperties implements Serializable {
    */
   public static final String LAKE_FORMATION_DB_NAME = "lakeformation.db-name";
 
+  /** Region to be used by the SigV4 protocol for signing requests. */
+  public static final String REST_SIGV4_SIGNER_REGION = "rest.sigv4.signing-region";
+
+  /** The service name to be used by the SigV4 protocol for signing requests. */
+  public static final String REST_SIGV4_SIGNING_NAME = "rest.sigv4.signing-name";
+
+  /** The default service name (API Gateway and lambda) used during SigV4 signing. */
+  public static final String REST_SIGV4_SIGNING_NAME_DEFAULT = "execute-api";
```

Comment:
Why no `rest.` prefix for this one?

Reply:
This is the default value, not a property that can be set.  The `execute-api` is used for api gateway and lambdas as the service name.

Reply:
Could we clarify that in the name? What about `API_GATEWAY_SIGV4_NAME`?

Reply:
The current naming follows the pattern of all the other default properties.  It's really not useful outside of the context of this class so maybe just make it private?

Reply:
That works for me, but I still think it should be renamed. This identifies API gateway, not REST right?